workflow.cwl

```diff
@@ -265,6 +265,52 @@ steps:
         source: "#validation_email/finished"
     out: [finished]
 
+
+## SYNTHETIC QUEUE SCORING
+##########################
+
+  determine_question:
+    run: determine_question.cwl
+    in:
+      - id: queue
+        source: "#get_docker_submission/evaluation_id"
+    out:
+      - id: question
+
+  scoring:
```

Comment:
This will basically be removed once we add in real data right?  We aren't going to continue scoring the synthetic data

Reply:
Right.  This was just for testing and for showing Mike how it can work for when we score with the real data during the Synthetic Validation phase.

Reply:
Yup. Feel free to merge when youre ready as the workflows for main eval queues pull from master